walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VCVerifiedIdSerializer.kt

```diff
@@ -3,15 +3,25 @@
 package com.microsoft.walletlibrary.verifiedid
 
 import com.microsoft.walletlibrary.did.sdk.credential.service.models.contracts.VerifiableCredentialContract
+import kotlinx.serialization.json.Json
+
+object VCVerifiedIdSerializer : VerifiedIdSerializer<String> {
```

Comment:
Don't we already have a string serializer?

Reply:
Removed and reused the existing string serializer and combined the methods in the same serializer class.